File: Copilot/tipos.java
public class tipos {
	public static double precioFinal(double precio, String descuentoTexto) {
		double descuento = Double.parseDouble(descuentoTexto);
		return precio - (precio * descuento / 100.0);
	}

	public static void main(String[] args) {
		double precioFinalResultado = precioFinal(50, "10");
		System.out.println(precioFinalResultado);
	}
}

File: Copilot/tipos.py
def precio_final(precio, descuento_texto):
	descuento = float(descuento_texto)
	return precio - (precio * descuento / 100)


if __name__ == "__main__":
	precio_final_resultado = precio_final(50, "10")
	print(precio_final_resultado)
